src/main/java/uk/gov/laa/ccms/caab/bean/validators/application/ApplicationSummaryValidator.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.laa.ccms.caab.bean.validators.application;
+
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import uk.gov.laa.ccms.caab.bean.ApplicationDetails;
+import uk.gov.laa.ccms.caab.bean.validators.AbstractValidator;
+import uk.gov.laa.ccms.caab.model.ApplicationSummaryDisplay;
+
+/**
+ * Validates the application summary details provided by the user.
+ */
+@Component
+public class ApplicationSummaryValidator extends AbstractValidator {
+
+  /**
+   * Determine if this validator can validate instances of the given class.
+   *
+   * @param clazz the class to check
+   * @return whether this validator can validate instances of the given class.
+   */
+  @Override
+  public boolean supports(Class<?> clazz) {
+    return ApplicationSummaryDisplay.class.isAssignableFrom(clazz);
+  }
+
+  /**
+   * Validate the given target object.
+   *
+   * @param target the target object to validate
+   * @param errors contextual state about the validation process
+   */
+  @Override
+  public void validate(Object target, Errors errors) {
+    ApplicationSummaryDisplay applicationSummary = (ApplicationSummaryDisplay) target;
```

Comment:
It doesn't do any validation?

Reply:
This will be required when i come back to the summary page when we can submit the application.
This will need to check whether all the sections are complete.